web/libs/editor/src/tags/object/TimeSeries.jsx

```diff
@@ -89,6 +90,9 @@ const TagAttrs = types.model({
   // visibilitycontrols: types.optional(types.boolean, false), // show channel visibility controls
   hotkey: types.maybeNull(types.string),
   sync: types.maybeNull(types.string),
+
+  // Color of the playback cursors (hex or any SVG-compatible color string)
+  cursorcolor: types.optional(types.string, "#1e90ff"),
```

Comment:
```suggestion
  cursorcolor: types.optional(types.string, "var(--color-neutral-inverted-surface)"),
```

@ricardoantoniocm what is the semantic default color we would use here for a playhead?

Reply:
We should use `--color-neutral-inverted-surface` in this instance.